### ISSUE TITLE
no such file or directory

### ISSUE THREAD

**Issue by gajus** [tbranyen/salita#7]
Getting this on a fresh node install.

```
curiosity:react-youtube-player gajus$ nvm install 4.2.1
######################################################################## 100.0%
WARNING: checksums are currently disabled for node.js v4.0 and later
Now using node v4.2.1 (npm v2.14.7)
curiosity:react-youtube-player gajus$ npm install salita -g
npm ERR! Darwin 15.0.0
npm ERR! argv "/Users/gajus/.nvm/versions/node/v4.2.1/bin/node" "/Users/gajus/.nvm/versions/node/v4.2.1/bin/npm" "install" "salita" "-g"
npm ERR! node v4.2.1
npm ERR! npm  v2.14.7
npm ERR! path /Users/gajus/.nvm/versions/node/v4.2.1/lib/node_modules/salita/node bin/salita
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod

npm ERR! enoent ENOENT: no such file or directory, chmod '/Users/gajus/.nvm/versions/node/v4.2.1/lib/node_modules/salita/node bin/salita'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/gajus/Documents/dev/gajus/react-youtube-player/npm-debug.log
```

**Comment by ljharb** [comment]
Oops, clearly I didn't test this well enough. Fixing!

**Comment by ljharb** [comment]
Published as v0.8.4
